src/main/resources/db/reference/modernised_darts_schema.sql

```diff
@@ -210,14 +210,35 @@
 --    add error_code and is_response_cleaned to external_object_directory
 --    add clip_id to media, annotation_document, transcription_document
 --    amend event_ts to not null on event
---
+--v63 add PK constraints to case_transcription_ae & hearing_transcription_ae
+--    add courthouse_code to daily_list
+--    amend transcription_commment.tra_id to be not null
+--v64 add retain_until_ts to annotation_document, case_document, media, transcription_document
+--    add last_modified_by/ts to annotation_document, case_document, transcription_document
+--    add is_deleted to court_case, media, transcription
+--    add update_retention to external_object_directory
+--    add is_retention_updated to court_case
+--    amend user_full_name and user_name on user_account to nullable
+--    amend checksum on annotation_document, case_document, transcription_document to nullable
+--    add retention_retries to court_case
+--    amend retain_until_ts on annotation_document, case_document, media, transcription_document to nullable
+--    add marked_for_manual_deletion to annotation_document, case_document, media, transcription_document
+--    add table object_hidden_reason 
+--    add fk and column ohr_id to annotation_document, case_document, media, transcription_document
+--    amend is_hidden to default to false on annotation_document, case_document, media, transcription_document
+--    amend is_deleted to default to false on annotation, court_case, media, transcription
+--    add osr_uuid on external_object_directory
+--    amend external_location on external_object_directory to nullable
+--    add enhanced_auditing to audit table
+--v65 add table audit_heritage
 
 
 -- List of Table Aliases
 -- annotation                  ANN
 -- annotation_document         ADO
 -- audit                       AUD
 -- audit_activity              AUA
+-- audit_heritage              AUH

```

Comment:
Do we need to have a corresponding sequences for these 2 new tables: `auh_seq` and `ohr_seq`?
If so, then also drop statements in the other file

Reply:
audit history is purely a copy of the legacy audit, hence no sequence to match auh_seq is envisaged.
ohr_seq has been added and the corresponding drop will be found in v23 of the drop script.